tests/unit/amo/test_installAddon.js

```diff
@@ -147,6 +149,7 @@ describe(__filename, () => {
     sinon.assert.calledWith(_addonManager.getAddon, addon.guid);
   });
 
+  // FIXME
```

Comment:
I think for these the test might have to be in a page component, rather than in `TestInstallButtonWrapper`. Although you might be able to get it to work by loading a different `addon` or `version` after rendering the component. That might cause the component to update.